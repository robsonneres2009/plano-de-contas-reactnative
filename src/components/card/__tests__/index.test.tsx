import React from "react";
import renderer from "react-test-renderer";
import Card from "..";
import { Text } from "react-native";

describe("Card", () => {
  it("Render component", () => {
    const tree = renderer
      .create(
        <Card key={1}>
          <Text>Card</Text>
        </Card>
      )
      .toJSON();
    expect(tree?.children?.length).toBe(1);
  });

  it("Render component snapshot", () => {
    const tree = renderer
      .create(
        <Card key={1}>
          <Text>Card</Text>
        </Card>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
