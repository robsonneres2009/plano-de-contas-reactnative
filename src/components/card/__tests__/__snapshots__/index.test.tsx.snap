// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Render component snapshot 1`] = `
<View
  style={
    {
      "backgroundColor": "#F0EDF5",
      "borderTopEndRadius": 28,
      "borderTopStartRadius": 28,
      "height": "100%",
      "marginTop": 10,
      "paddingHorizontal": 24,
      "paddingVertical": 16,
      "width": "100%",
    }
  }
>
  <Text>
    Card
  </Text>
</View>
`;
